### ISSUE TITLE
Grace period for updating scores

### ISSUE THREAD

**Issue by marisademeglio** [IDPF/epubtestweb#58]
After a test book update, accessibility evaluators need time to update their scores. So, either maintain 2 versions of the test suite, or save a static view of the old scores. The old scores should be displayed publicly until the evaluation has been updated.

**Comment by marisademeglio** [comment]
No longer applicable
